lib/github_s.mli

```diff
@@ -1001,6 +1001,124 @@ module type Github = sig
+      user:string -> 
+      repo:string -> 
+      name:string -> 
+      unit -> 
+      Github_t.label Response.t Monad.t
+    (** [get ~user ~repo ~name ()] gets the label [name] from the
+        repo [user]/[repo]. *)
+
+    val create :
+      ?token:Token.t ->
+      user:string -> 
+      repo:string -> 
+      label:Github_t.new_label -> 
+      unit -> 
+      Github_t.label Response.t Monad.t
+    (** [create ~user ~repo ~label ()] creates the label [label] in the
+        repo [user]/[repo]. *)
+
+    val update :
+      ?token:Token.t ->
+      user:string ->
+      repo:string ->
+      name:string ->
+      label:Github_t.new_label -> 
+      unit ->
+      Github_t.label Response.t Monad.t
+    (** [update ~user ~repo ~name ()] updates the label [name] in the
+        repo [user]/[repo]. *)
+
+    val delete :
+      ?token:Token.t ->
+      user:string ->
+      repo:string ->
+      name:string ->
+      unit ->
+      unit Response.t Monad.t
+    (** [delete ~user ~repo ~name ()] deletes the label [name] in the
+        repo [user]/[repo] *)
+
+    val for_repo_issue :
+      ?token:Token.t ->
+      user:string ->
+      repo:string ->
+      num:int ->
+      unit ->
+      Github_t.label Stream.t
+    (** [for_repo_issue ~user ~repo ~num ()] is a stream of all labels
+        applied to issue [num] in the repo [user]/[repo]. *)
+
+    val add_to_issue :
+      ?token:Token.t ->
+      user:string ->
+      repo:string ->
+      num:int ->
+      labels:Github_t.new_labels ->
+      unit -> 
+      Github_t.label list Response.t Monad.t
+    (** [add_to_issue ~user ~repo ~num ~labels ()] adds the labels [labels]
+        to issue [num] in the repo [user]/[repo]. *)
+
+    val remove_from_issue :
+      ?token:Token.t ->
+      user:string ->
+      repo:string ->
+      num:int ->
+      name:string ->
+      unit ->
+      unit Response.t Monad.t
+    (** [remove_from_issue ~user ~repo ~num ~name ()] adds the label [name]
+        from the issue [num] in the repo [user]/[repo]. *)
+
+    val replace_all_on_issue :
+      ?token:Token.t ->
+      user:string ->
+      repo:string ->
+      num:int ->
+      labels:Github_t.new_labels ->
+      unit ->
+      Github_t.label list Response.t Monad.t
+    (** [replace_all_on_issue ~user ~repo ~num ~labels ()] replaces the labels on
+        issue [num] in the repo [user]/[repo] with those provided in [labels]. *)
+
+    val remove_all_from_issue :
+      ?token:Token.t ->
+      user:string ->
+      repo:string ->
+      num:int ->
+      unit ->
+      unit Response.t Monad.t
+    (** [remove_all_from_issue ~user ~repo ~num ()] remooves all labels
+        from the issue [num] in the repo [user]/[repo]. *)
+
+     val for_milestone :
+      ?token:Token.t ->
+      user:string ->
+      repo:string ->
+      num:int -> 
+      unit ->
+      Github_t.label Stream.t
+    (** [for_repo ~user ~repo ()] is a stream of all labels in repo
+        [user]/[repo] *)
```

Comment:
This doc text is for `for_repo` not `for_milestone` (but see disaggregation comments above).